src/components/SurveyComponents/SurveyStep.tsx

```diff
@@ -0,0 +1,8 @@
+import { ReactNode } from 'react'
+
+interface propTypes {
+  children?: ReactNode
+}
+export function SurveyStep({ children }: propTypes) {
```

Comment:
it is a convention that the Props interface has the same name of the component + Props, i.e. in this case it should be SurveyStepProps